### ISSUE TITLE
Clarify project status on the website and in the repo

### ISSUE THREAD

**Issue by clayrat** [miniboxing/miniboxing-plugin#252]
It seems like the plugin is not developed anymore - were there any plans to utilize it in 2.12 or Dotty?

**Comment by VladUreche** [comment]
Thanks for the ping @clayrat! Indeed, I have found less and less time to work on miniboxing. Do you need it ported to 2.12 for a project?

Re. dotty, it will have an improved version of specialization based on the dotty linker, and you can see a paper on it by @DarkDimius here: https://infoscience.epfl.ch/record/217276?ln=en

**Comment by clayrat** [comment]
No, nothing critical, I was just curious about porting old examples, e.g. https://github.com/tixxit/optimistic-spec-article

I guess, let's wait for Dotty then :)

**Comment by VladUreche** [comment]
I've updated the issue description to clarify the project status, so anyone looking to use it can contact me if they need support for 2.12.

**Comment by Sciss** [comment]
Is the project dead thus? What about Scala 2.13?

**Comment by VladUreche** [comment]
It just needs resuscitation from a few kind-hearted contributors ;)

**Comment by SethTisue** [comment]
fyi @jorokr21. I've only seen scattered remarks about why the Scala 3 team hasn't pursued miniboxing further.

One such remark is Vlad's remark above about Dmitry's plans.

I also found https://twitter.com/alexelcu/status/1365162988362203136, which isn't very specific, but does kinda sound like @alexelcu asked somebody important about it.

Miniboxing isn't mentioned at https://contributors.scala-lang.org/t/status-of-specialization-in-scala-3/4628/10 (“Status of specialization in Scala 3”), but that thread lacks any real full response from an insider. There's also https://contributors.scala-lang.org/t/on-demand-specialization/4350/2 where Martin says "The best bet is probably to compile and specialize from Tasty"

**Comment by VladUreche** [comment]
Thanks everyone for your interest in miniboxing and thank you, @SethTisue, for following up.

TL;DR: As much as I would love to see miniboxing to Scala 3, this will need someone rolling up their sleeves and implementing it (and I don't have the resources to do this atm). The solution @odersky suggested in https://contributors.scala-lang.org/t/on-demand-specialization/4350/2 is very attractive, and I don't see any obvious drawbacks.

If anyone wants to give this a stab, please read the papers at http://scala-miniboxing.org/mbox/index.html and http://scala-miniboxing.org/ldl/index.html and ask questions here -- I'm following the thread and will answer to the best of my knowledge.

**Comment by SethTisue** [comment]
see also https://github.com/lampepfl/dotty/issues/15532
